Utils/Dataflow/017_adjustMetadata/adjustMetadata.py

```diff
@@ -28,6 +28,45 @@
     sys.exit(1)
 
 
+PHYS_CATEGORIES_HASHTAGS_MAP = {
+    'BPhysics': ['charmonium', 'jpsi', 'bs', 'bd', 'bminus', 'bplus',
+                 'charm', 'bottom', 'bottomonium', 'b0'],
+    'BTag': ['btagging'],
+    'Diboson': ['diboson', 'zz', 'ww', 'wz', 'wwbb', 'wwll'],
+    'DrellYan': ['drellyan', 'dy'],
+    'Exotic': ['exotic', 'monojet', 'blackhole', 'technicolor',
+               'randallsundrum', 'wprime', 'zprime', 'magneticmonopole',
+               'extradimensions', 'warpeded', 'randallsundrum',
+               'contactinteraction', 'seesaw'],
+    'GammaJets': ['photon', 'diphoton'],
+    'Higgs': ['whiggs', 'zhiggs', 'mh125', 'higgs', 'vbf', 'smhiggs',
+              'bsmhiggs', 'chargedhiggs'],
+    'Minbias': ['minbias'],
+    'Multijet': ['dijet', 'multijet', 'qcd'],
+    'Performance': ['performance'],
+    'SingleParticle': ['singleparticle'],
+    'SingleTop': ['singletop'],
+    'SUSY': ['bino', 'susy', 'pmssm', 'leptosusy', 'rpv', 'mssm'],
+    'Triboson': ['triplegaugecoupling', 'triboson', 'zzw', 'www'],
+    'TTbar': ['ttbar'],
+    'TTbarX': ['ttw', 'ttz', 'ttv', 'ttvv', '4top', 'ttww'],
+    'Upgrade': ['upgrad'],
+    'Wjets': ['w'],
+    'Zjets': ['z']}
+
+
+PHYS_CATEGORIES_TASKNAME_MAP = {
```

Comment:
It is more like "PHYS_SHORT_MAP" (or "PH_SORT_MAP"), then "TASKNAME". Just to avoid misunderstanding, or else someone might believe we really check the whole taskname, while in fact it is only "physics short" field.

Reply:
Done.